### PR TITLE
Add missing log param

### PR DESCRIPTION
**Pull request by FSchumacher**
Found two places, where either a placeholder or a parameter was missing.

### CODE REVIEW

File: activemq-broker/src/main/java/org/apache/activemq/broker/region/Queue.java [hunk 1]
```diff
@@ -2100,7 +2100,7 @@ private PendingList doPageInForDispatch(boolean force, boolean processExpired, i
                                 ConnectionContext connectionContext = createConnectionContext();
                                 dropMessage(ref);
                                 if (gotToTheStore(ref.getMessage())) {
-                                    LOG.debug("Duplicate message {} from cursor, removing from store", this, ref.getMessage());
+                                    LOG.debug("{}, duplicate message {} from cursor, removing from store", this, ref.getMessage());
```
Comment (coheigea) [inline activemq-broker/src/main/java/org/apache/activemq/broker/region/Queue.java]: I think here we could just remove logging this from the log statement

Comment (jbonofre) [inline activemq-broker/src/main/java/org/apache/activemq/broker/region/Queue.java]: Can you please use:

```
LOG.debug("Duplicate message {] from cursor, removing from store", ref.getMessage());
```

`this` is useless here.

Comment (FSchumacher) [reply activemq-broker/src/main/java/org/apache/activemq/broker/region/Queue.java]: Right, the problem was, that `this` was used as first parameter in the log format string that had only one parameter. So the second  parameter `ref.getMessage()` was silently dropped.
As I didn't know, if `this` was needed for your analysis of the log messages, I added a second parameter placeholder.
I will try to update this PR.

File: activemq-broker/src/main/java/org/apache/activemq/broker/TransportConnection.java [hunk 2]
```diff
@@ -581,7 +581,7 @@ public Response processMessageAck(MessageAck ack) throws Exception {
         if (consumerExchange != null) {
             broker.acknowledge(consumerExchange, ack);
         } else if (ack.isInTransaction()) {
-            LOG.warn("no matching consumer, ignoring ack {}", consumerExchange, ack);
+            LOG.warn("no matching consumer for {}, gnoring ack {}", consumerExchange, ack);
```
Comment (coheigea) [inline activemq-broker/src/main/java/org/apache/activemq/broker/TransportConnection.java]: gnoring -> ignoring

**Comment by coheigea** [comment]
@jbonofre This PR can be closed, as it's superceded by https://github.com/apache/activemq/pull/304

**Pull request closed without merging**